src/app/select-teams/select-teams.component.css

```
@@ -0,0 +1,64 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.subContainer {
+  display: flex;
+  flex-direction: row;
+}
+
+.selectedContainer {
+}
+
+.submitButtonContainer {
+  display: flex;
+  justify-content: center;
+  align-self: flex-end;
+  width: 300px;
+  margin-bottom: 30px;
+}
+
+.submitButton {
+  height: 30px;
+  width: 200px;
+}
+
+.optionContainer {
+  display: flex;
+  flex-direction: column;
+  display: inline-block;
+  border: solid 1px;
+  border-color: black;
+}
+
+.leagueContainer {
+  display: flex;
+  flex-direction: row;
+}
+
+.small {
+  height: 120px;
+  width: 120px;
+  border: solid 1px;
+  border-color: black;
+  margin: 15px;
+}
+
+.small:hover {
+  height: 125px;
+  width: 125px;
+  border: solid 2px;
+  border-color: #0000ff;
+  margin: 10px auto 15px;
+}
```

scss使っちゃう？（使ったことあったっけ？）
拡張子.scssにして読み込んでるところをも.scssに合わせるだけでいけるはず

---

あんま機能使いまくったりしなくていいと思うけど疑似要素のところだけまとめて書けると見やすいかなって思った

```suggestion
.small {
  height: 120px;
  width: 120px;
  border: solid 1px;
  border-color: black;
  margin: 15px;
  
  &:hover {
    height: 125px;
    width: 125px;
    border: solid 2px;
    border-color: #0000ff;
    margin: 10px auto 15px;
  }
}
```

---

scss使ったことなかったです。
まだ機能とか全然分かってないですが、一旦適用してみました。